src/components/pages/LandingPage.vue

```diff
@@ -1,24 +1,41 @@
 <script>
-    import { defineComponent } from 'vue';
+import { defineComponent } from 'vue';
 import TheHeader from '@/components/views/TheHeader.vue';
-import TopNavbar from '@/components/features/TopNavbar.vue';
-import SignSection from '@/components/features/SignSection.vue';
+import TopNavbar from '@/components/features/Header/TopNavbar.vue';
+import SignSection from '@/components/features/Header/SignSection.vue';
 import TheContainer from '@/components/common/TheContainer.vue';
 import WelcomeSection from '@/components/views/WelcomeSection.vue';
-import MovieCard from '@/components/views/MovieCard.vue';
 import MoviesSummary from '@/components/views/MoviesSummary.vue';
+import { getAllMovies } from '@/api/service/movies';
 import ScreeningsFilters from '@/components/views/ScreeningsFilters.vue';
+
 export default defineComponent({
   components: {
     TheHeader,
     TopNavbar,
     SignSection,
     TheContainer,
     WelcomeSection,
-    MovieCard,
     MoviesSummary,
     ScreeningsFilters
-}
+  },
+  data() {
+    return {
+      movies: [],
+      filteredMovies: [],
+    };
+  },
+  async mounted() {
+    try {
+      const response = await getAllMovies();
+      this.movies = response.data;
+    } catch {
+      this.$router.push({ name: 'NoMatch' });
+    }
+    for (let i=0; i<3; i++) {
+      this.filteredMovies.push(this.movies[i])
```

Comment:
Maybe you can move it to computed and try to handle with some array method? 🙂

Reply:
I already did it on the next branch -- will come with the next PR 😃